src/common/components/Feed/index.tsx

```diff
@@ -0,0 +1,49 @@
+import Group from '../Group';
+import Icon from '../Icon';
+import Image from '../Image';
+import Like from '../Like';
+import Text from '../Text';
+
+interface FeedProps {
+  initialState: boolean;
+  title: string;
+  image: string;
+  body?: string;
+  likes: object[];
+  comments: object[];
+}
+
+const Feed = ({
+  initialState = false,
+  title,
+  image,
+  body,
+  likes,
+  comments,
+}: FeedProps) => {
+  return (
+    <Group direction="columns" spacing="md">
+      <Text size="xlarge" strong={true}>
+        {title}
+      </Text>
+      {Boolean(image) === true ? (
```

Comment:
```suggestion
{image && <Image />
```

이런 식으로 쓸 수도 있을 것 같아요!

그리고 이미지 최대 값을 정해두는 건 어떨까요? 가로 너비 같은 경우엔 layout이 있다 하더라도, 세로의 경우에는 제한이 없어 보이네요!

Reply:
저도 이미지의 세로 길이가 너무 길 경우 불편할 수 있을 것 같아 어느 정도 제한을 두는게 맞다고 생각했어요! 성민님이 공유해주신 `aspect-video` 속성을 사용해 수정했습니다! 

`메인 페이지`와 `마이 페이지`에서는 이미지를 `16:9`로 보여주고, `상세 페이지`에서는 이미지 전체를 보여줘도 좋을 것 같아, `imgAspect`에 boolean 값을 설정할 수 있도록 해 `true`일 때 `aspect-video`를 적용할 수 있도록 했습니다!